### PR TITLE
Max execution time

### PR DESCRIPTION
**Pull request by ccsuperstar**
Related to issue #119 

The html reports with Codeception 4 have an execution time format of the form 198.165s
The html reports with Codeception 5 have an execution time format of the form 22:45.165

The regex matches both versions

Added maxSuiteTime support for the final xml report (the execution time of a suite is the longest run time among all parallel runs)
`max($this->suiteDuration[$suiteName])`

Added maxTime support for the final html report (the total execution time is the longest run time among all parallel runs)
`$this->executionTime[] = (string)$matches['timesum'];`
`$executionTime = max($this->executionTime);`

I am looking for feedbacks, I think everything can be improved!

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @vansari

**Comment by vansari** [comment]
@ccsuperstar Thank you for your contribution. Please do not forget to write a test for your changes.

**Comment by ccsuperstar** [comment]
> @ccsuperstar Thank you for your contribution. Please do not forget to write a test for your changes.

yes I will do it, then I guess I have to do tests for codeception 5 and for the previous versions

File: src/Merger/HtmlReportMerger.php [hunk 1]
```diff
@@ -342,4 +379,38 @@ private function updateButtons(DOMDocument $dstFile): void
             $table->setAttribute('id', "stepContainer" . $n);
         }
     }
+
+    private function sumTime(string $time1, string $time2): string
+    {
+        $times = array($time1,  $time2);
```
Comment (vansari) [inline src/Merger/HtmlReportMerger.php:385]:
```suggestion
        $times = [$time1,  $time2];
```

**Review by vansari — CHANGES_REQUESTED** [review]

**Review by vansari — APPROVED** [review]

**Pull request merged**